### PR TITLE
Update to harfbuzz 2.5.0.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This change is [Reviewable](https://reviewable.io/reviews/servo/rust-harfbuzz/149)

### CODE REVIEW

**Comment by jdm** [comment]
Linux build is unhappy.

**Comment by waywardmonkeys** [comment]
@jdm Do you know offhand what version of `gcc` is being used on the Linux builds?

From the Harfbuzz 2.5.0 release notes:

> This release does not include much functional changes, but includes major
> internal code-base changes. We now require C++11. Support for gcc 4.8
> and earlier has been dropped.

**Comment by jdm** [comment]
https://docs.travis-ci.com/user/languages/c/#c11c11-and-beyond-and-toolchain-versioning

**Comment by waywardmonkeys** [comment]
The Linux build is happy now (using gcc 5).

I could probably simplify the changes to `.travis.yaml` if desired.

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,16 @@ matrix:
...
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - g++-5
+      env:
+         - PREFIX=/usr/local
+         - CC=gcc-5
+         - CXX=g++-5"
```
Comment (jdm) [inline .travis.yml]: Extra `"` shouldn't be here, and in two other uses of CXX.

**Comment by jdm** [comment]
Once those are fixed this is r=me.

**Comment by waywardmonkeys** [comment]
@jdm I think this is ready for you to approve again!

**Pull request merged**